packages/trader/src/AppV2/Components/Guide/guide.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button } from '@deriv-com/quill-ui';
+import { LabelPairedPresentationScreenSmRegularIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+import { CONTRACT_LIST } from 'AppV2/Utils/trade-types-utils';
+import DefinitionModal from './definition-modal';
+import DescriptionModal from './decription-modal';
+
+type TGuide = {
+    is_minimalistic_look?: boolean;
+};
+
+const Guide = ({ is_minimalistic_look = false }: TGuide) => {
```

Comment:
```suggestion
const Guide = ({ is_minimalistic = false }: TGuide) => {
```
or `{ has_label = true }` to make it minimalistic when has_label={false}.
also, this is a matter of taste, up to you :)